backend/src/test/resources/application.properties

```diff
@@ -19,6 +19,9 @@ puris.dtr.idp.edc-client.id=${PURIS_DTR_IDP_EDC-CLIENT_ID:FOSS-DTR-CLIENT}
 puris.dtr.idp.edc-client.secret.alias=${PURIS_DTR_IDP_EDC-CLIENT_SECRET_ALIAS:test-alias}
 puris.dtr.idp.puris-client.id=${PURIS_DTR_IDP_PURIS-CLIENT_ID:FOSS-DTR-CLIENT}
 puris.dtr.idp.puris-client.secret=${PURIS_DTR_IDP_PURIS-CLIENT_SECRET:test}
+
+puris.erpadapter.url=${PURIS_ERPADAPTER_URL:http://my-erpadapter:8080}
```

Comment:
Not adding all properties leads to failure of test cases: 
- `PurisApplicationTests`
- `ApiKeyTests`

`Caused by: java.lang.IllegalArgumentException: Could not resolve placeholder 'puris.erpadapter.authkey' in value "${puris.erpadapter.authkey}"`

Beside that we can already introduce a property `enabled` to ignore empty properties in that case. Likely it makes sense to create a Class `ErpAdapterServiceConfiguration` similar to `DtrSecurityConfiguration`

Reply:
Ooops, sorry, corrected the missing properties there. Also added that config class